sites/hcinnovationgroup.com/server/templates/website-section/top-100.marko

```diff
@@ -66,17 +66,15 @@ $ const { id, alias, name, pageNode } = data;
                       flush=true
                     >
                     <@body>
-                      <@text modifiers=["teaser"] >
-                        <div class="d-flex">
-                          <div class="rank">${node.rank}</div>
-                          <div class="company flex-grow-1">
-                            <marko-web-content-short-name modifiers="title" tag="h6" obj=node link={ attrs: linkAttrs }>
-                              <@link href=node.siteContext.url />
-                            </marko-web-content-short-name>
-                            ${node.city}, ${node.state}
-                          </div>
-                          <div class="revenue">${formatUsCurrency(node.revenueCurrent)}</div>
+                      <@text>
+                        <div class="rank">${node.rank}</div>
+                        <div class="company">
+                          <marko-web-content-short-name class="name" tag="h6" obj=node link={ attrs: linkAttrs }>
```

Comment:
Using the `blockName` from above, remove the `class="name"` attribute and add `block-name=blockName`

This will automatically create a `node__content-short-name` element